### ISSUE TITLE
Accessibility findings and duplicate CSS selector in dependency-check-report.html

### ISSUE THREAD

**Issue by proxity** [dependency-check/DependencyCheck#7618]
SonarQube demands 3 fixes to some HTML elements of dependency-check-report.html which is produced by the current version dependency-check 12.1.1. I know sometimes it's really tedious to work on accessibility issues, but at least at our company we try to do the best to serve all possible users. Would be great if this could be fixed.

Rated as a major bug is: Add "lang" and/or "xml:lang" attributes to the `"<html>"` element at the beginning of the file.
Reason: `"<html>"` element should have a language attribute due to accessibility issues.

Rated as a minor bug is: Add a description to this `<table id="summaryTable" class="lined">` which is found towards the end of the file.
Reason: `"<table>"` tags should have a description due to accessibility issues.

Rated as a major code-smell is: duplicate selector ".hidden", actually this CSS element is specified two time
Reason: selectors should not be duplicated, possible copy-paste mistake
